packages/lending/src/adapter.ts

```diff
@@ -500,7 +500,7 @@ export class Adapter extends common.Web3Toolkit {
         if (flashLoanRepayLogic) output.logics.push(flashLoanRepayLogic);
 
         if (withdrawalToken.isNative && Number(withdrawal.amount) > 0) {
-          const wrapNativeLogic = await apisdk.protocols.utility.newWrappedNativeTokenLogic({
+          const wrapNativeLogic = apisdk.protocols.utility.newWrappedNativeTokenLogic({
```

Comment:
Can you help remove other `await` before `newWrappedNativeTokenLogic`?

Reply:
sure, there are two ones in api package, will remove them after this mr